syntaxes/omg.idl.tmLanguage.json

```diff
@@ -0,0 +1,567 @@
+{
```

Comment:
Could you use 4-spaces instead of tabs? The rest of documents are using that convention